tapchannel/commitment.go

```diff
@@ -93,19 +94,19 @@ func ComputeView(ourBalance, theirBalance uint64,
 	localHtlcIndex := make(map[uint64]lnwallet.AuxHtlcDescriptor)
 	remoteHtlcIndex := make(map[uint64]lnwallet.AuxHtlcDescriptor)
 
-	for _, entry := range original.AuxOurUpdates() {
+	for _, entry := range original.Updates.Local {
```

Comment:
This change and the new parameter in payment bandwidth should probably be in the same commit as the lnd bump commit, if we want each commit to compile individually (which at least in lnd we want but I think we haven't ported over that CI step to `tapd` yet).